packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -98,4 +117,48 @@ contract SuperchainWETH is WETH98, IERC7802, ISemver {
         return _interfaceId == type(IERC7802).interfaceId || _interfaceId == type(IERC20).interfaceId
             || _interfaceId == type(IERC165).interfaceId;
     }
+
+    /// @notice Sends ETH to some target address on another chain.
+    /// @param _to       Address to send tokens to.
+    /// @param _chainId  Chain ID of the destination chain.
+    function sendETH(address _to, uint256 _chainId) external payable returns (bytes32 msgHash_) {
+        if (_to == address(0)) revert ZeroAddress();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            revert NotCustomGasToken();
+        }
+
+        IETHLiquidity(Predeploys.ETH_LIQUIDITY).burn{ value: msg.value }();
+
+        msgHash_ = IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).sendMessage({
+            _destination: _chainId,
+            _target: address(this),
+            _message: abi.encodeCall(this.relayETH, (msg.sender, _to, msg.value))
+        });
+
+        emit SendETH(msg.sender, _to, msg.value, _chainId);
+    }
+
+    /// @notice Relays ETH received from another chain.
+    /// @param _from       Address of the msg.sender of sendETH on the source chain.
+    /// @param _to         Address to relay ETH to.
+    /// @param _amount     Amount of ETH to relay.
+    function relayETH(address _from, address _to, uint256 _amount) external {
+        if (msg.sender != Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert Unauthorized();
+
+        (address crossDomainMessageSender, uint256 source) =
+            IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).crossDomainMessageContext();
+
+        if (crossDomainMessageSender != address(this)) revert InvalidCrossDomainSender();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            revert NotCustomGasToken();
+        }
```

Comment:
@tynes regarding this comment: https://github.com/ethereum-optimism/design-docs/pull/146#issuecomment-2485175933

I think we should we revert and let the message expire (user get's their ETH back on origin)
being true to the intent of expecting ETH on destination is more important than forcing successful execution and transferring an ERC20.
wdyt?

Reply:
I lean more towards transferring `SuperchainWETH` to the recipient on the destination. I think it would be a bad experience to make the user wait for the message to expire before they can get their ETH back on origin. I also think it is intuitive if the native asset on the chain is not ETH to receive `SuperchainWETH`. Additionally, based on this https://github.com/ethereum-optimism/specs/issues/460 the devx is not great for implementing rollbacks and we should avoid them if possible

Reply:
While there is extra testing required, I lean towards just skipping the unwrapping if its a custom gas token chain. This would result in good ux for sending ether to a custom gas token chain